app/helpers/manage_metadata_helper.rb

```diff
@@ -1,12 +1,15 @@
 # :nodoc:
 module ManageMetadataHelper
   def display_entry_id(metadata, type)
-    blank_short_name = type == 'draft' ? '<Blank Short Name>' : 'New Collection'
-    short_name = metadata['ShortName'] || blank_short_name
+    short_name = if type.include? 'collection'
```

Comment:
Right now it is either coming from `resource_name` or `'collections'`. I've already changed this method some for MMT-1117, I'll include downcase in that ticket